package com.oztechan.ccc.common.core.network.error

internal class TimeoutException(cause: Throwable) : Throwable(cause)
